.gitignore

```diff
@@ -2,6 +2,7 @@
 __pycache__/
 *.py[cod]
 *$py.class
+.DS_Store
```

Comment:
should be in a global gitignore, refer to: https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore